src/celeritas/em/UrbanMscParams.cc

```diff
@@ -219,6 +219,10 @@ UrbanMscParams::calc_material_data(MaterialView const& material_view)
 
     double const zeff = material_view.zeff();
 
+    // Linear+quadratic parameters for the step minimum calculation
+    data.stepmin_coeff[0] = 1e3 * 27.725 / (1 + 0.203 * zeff);
```

Comment:
This line was simply moved from earlier. Digging back into the history, I moved the constant here from `UrbanMscHelper::calc_step_min` in 8e7e7eef8, and it had been there since the initial implementation. It's the same as the `1e-3` factor in `G4UrbanMscModel::ComputeStepmin`:

https://github.com/Geant4/geant4/blob/84a556a9dca683c2d541394a795642ecb4c07a3d/source/processes/electromagnetic/standard/include/G4UrbanMscModel.hh#L246

which of course doesn't have any provenance...

Since those values are hardcoded in the urban scattering model, leaving the multiplication factor separate makes it a little easier to figure out where the values came from... if we ever discover an original source though, we should add a reference and factor in the 1e3.